docs/api/compas.datastructures.VolMesh.rst

```diff
@@ -0,0 +1,240 @@
+******************************************************************************
+VolMesh
+******************************************************************************
+
+.. currentmodule:: compas.datastructures
+
+.. autoclass:: VolMesh
+
+Methods
+=======
+
+Constructors
+------------
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    ~VolMesh.from_json
+    ~VolMesh.from_obj
+    ~VolMesh.from_vertices_and_cells
+
+
+Conversions
+-----------
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    ~VolMesh.cell_to_mesh
+    ~VolMesh.cell_to_vertices_and_faces
+    ~VolMesh.to_json
+    ~VolMesh.to_obj
+    ~VolMesh.to_vertices_and_cells
+
+
+Builders and Modifiers
+----------------------
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    ~VolMesh.add_cell
+    ~VolMesh.add_halfface
+    ~VolMesh.add_vertex
+    ~VolMesh.delete_cell
+    ~VolMesh.delete_vertex
+    ~VolMesh.remove_unused_vertices
+
+
+Accessors
+---------
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    ~VolMesh.cell_sample
+    ~VolMesh.cells
+    ~VolMesh.cells_on_boundaries
+    ~VolMesh.cells_where
+    ~VolMesh.cells_where_predicate
+    ~VolMesh.edge_sample
+    ~VolMesh.edges
+    ~VolMesh.edges_where
+    ~VolMesh.edges_where_predicate
+    ~VolMesh.face_sample
+    ~VolMesh.faces
+    ~VolMesh.faces_where
+    ~VolMesh.faces_where_predicate
+    ~VolMesh.halffaces
+    ~VolMesh.halffaces_on_boundaries
+    ~VolMesh.vertex_sample
+    ~VolMesh.vertices
+    ~VolMesh.vertices_on_boundaries
+    ~VolMesh.vertices_where
+    ~VolMesh.vertices_where_predicate
+
+
+Attributes
```

Comment:
For this tweaked classes, the `attributes` section can be quite confusing with the normal python attributes. How about we name this `Custom attributes`?

Reply:
was thinking of splitting up the sections into "Vertex Attributes", "Edge Attributes", ...

Reply:
a bit like the sections in the code itself

Reply:
good idea 👍